README.md

```diff
@@ -1,4 +1,30 @@
 # ThunderDome
-## Two Goblins Enter One Goblin Leaves ##
+## Two Monsters Enter One Monster Leaves ##
 
 This is a stripped down D&D 5e combat simulator that I am making for practice.
+
+## Build and Run
+
+To build and run this project, first clone it from Github and navigate to the root of the project.  Open a terminal and run the following:
+
+```sh
+# Build
+dotnet build ThunderDome/
+
+# Rin
+dotnet run --project ThunderDome/ThunderDome/ThunderDome.csproj
+```
+
+Other things to consider
+
+Based on this [article](https://blog.markvincze.com/automated-portable-code-style-checking-in-net-core-projects/ )
+
+```sh
+dotnet add ThunderDome/ThunderDome/ThunderDome.csproj package StyleCop.Analyzers
```

Comment:
I wanted to enable basic code linting, not sure if this is the best way so ask your class.  However, now when you build you will get a boat load of warning.  It will be overwhelming at first but it is going to give you code style suggestions. You don't have to take them but most of the time they are good suggestions.